provisioning/install-precice.sh

```diff
@@ -14,10 +14,10 @@ fi
     cd precice
     git pull
     mkdir -p build && cd build/
-    cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTING=OFF -Wno-dev ..
+    cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_TESTING=OFF -Wno-dev ..
```

Comment:
To turn the debug log on.
Is there a reason to include debug info?

```suggestion
    cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo -DPRECICE_RELEASE_WITH_DEBUG_LOG=ON -DBUILD_TESTING=OFF -Wno-dev ..
```

Reply:
This originates from this (maybe outdated) suggestion: https://github.com/precice/vm/issues/41

Reply:
I think the intention was always that we might want to use the VM for debugging some cases involving difficult-to-install solvers.